### PR TITLE
[kubeflow 1.8] Update kserve manifests from v0.11.0

### PR DESCRIPTION
**Pull request by DnPlas**
Sync kserve/kserve tag v0.11.0 manifests

WG leads @yuzisun
WG liaison @helloericsf 
cc: @annajung @jbottum

### CODE REVIEW

File: contrib/kserve/kserve/kserve_kubeflow.yaml [hunk 1]
```diff
@@ -20110,7 +20630,8 @@ apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
   annotations:
-    cert-manager.io/inject-ca-from: kubeflow/serving-cert
+    cert-manager.io/inject-ca-from: kserve/serving-cert
```
Comment (annajung) [inline contrib/kserve/kserve/kserve_kubeflow.yaml:20633]: There are several places where annotation is `cert-manager.io/inject-ca-from: kserve/serving-cert`, but should be `cert-manager.io/inject-ca-from: kubeflow/serving-cert`

**Comment by annajung** [comment]
Verified that changing back to kubeflow successfully creates inference service

Also, the Kserve version should be updated in the `./contrib/kserve/tests/requirements.txt` to [0.11](https://pypi.org/project/kserve/0.11.0/)

**Comment by DnPlas** [comment]
> Verified that changing back to kubeflow successfully creates inference service
> 
> Also, the Kserve version should be updated in the `./contrib/kserve/tests/requirements.txt` to [0.11](https://pypi.org/project/kserve/0.11.0/)

ACK, I used the script as is. I can definitely make those changes manually, but I wonder if kserve wants to modify this in their own repo to prevent kubeflow/manifests diverging from kserve/kserve. 
In these scenarios what's the recommended way? @annajung

**Comment by annajung** [comment]
Maybe start with an issue to kserve/kserve repo, to notify them of this and to understand when they'll be able to provide new release with the fix? Not sure if this will require patch release, but I would assume it would? cc @yuzisun 

I would say, base on their timeline and kubeflow release timeline, we should decide how to move forward. If the timeline doesn't align, then I would suggest manual fix for now so that it doesn't block the release schedule and can be updated when we cut future RCs.

**Comment by DnPlas** [comment]
> 

Yeah, I agree. If the dates don't align with kserve by the time all the other components are synced, we can proceed and make the change manually. I'll open an issue and ping the team. Thanks @annajung.

**Comment by yuzisun** [comment]
@DnPlas @annajung Can you help update the kserve_kubeflow.yaml ? we have fixed the 0.11 [kubeflow release artifact](https://github.com/kserve/kserve/releases/download/v0.11.0/kserve_kubeflow.yaml).

File: contrib/kserve/kserve/kserve_kubeflow.yaml [hunk 2]
```diff
@@ -20087,9 +20607,9 @@ metadata:
   name: serving-cert
   namespace: kubeflow
 spec:
-  commonName: kserve-webhook-server-service.kubeflow.svc
+  commonName: kserve-webhook-server-service.kserve.svc
   dnsNames:
-    - kserve-webhook-server-service.kubeflow.svc
+    - kserve-webhook-server-service.kserve.svc
```
[the remark this answers is not in the document]
Comment (DnPlas) [reply contrib/kserve/kserve/kserve_kubeflow.yaml]: Resolving, see my [other comment](https://github.com/kubeflow/manifests/pull/2504#discussion_r1299977451)

File: contrib/kserve/kserve/kserve_kubeflow.yaml (hunk 1 above)
Comment (DnPlas) [reply contrib/kserve/kserve/kserve_kubeflow.yaml]: Resolving, see my [other comment](https://github.com/kubeflow/manifests/pull/2504#discussion_r1299977451)

**Comment by DnPlas** [comment]
Updated and ready for approval @kimwnasptd

**Pull request merged**